eclib/Jslh.ec

```diff
@@ -0,0 +1,37 @@
+require import CoreInt JWord.
+
+abstract theory SLH.
+  type msf_t.
+  op of_int : int -> msf_t.
+
+  abbrev [-printing] init_msf : msf_t = of_int 0.
+
+  (* FIXME ec *)
+  (* abbrev [-printing] move_msf (msf:msf_t) : msf_t = msf. *)
```

Comment:
This is solved in the main branch of EasyCrypt. Should I change this?

Reply:
I’d rather wait for the next release of EasyCrypt.